framework/Web/UI/TTemplate.php

```diff
@@ -441,6 +442,32 @@ protected function configureProperty($component, $name, $value)
 				}
 			}
 			$setter = 'set' . $name;
+			if (method_exists($component, $setter)) {
```

Comment:
Should be updated to be behavior aware with #825

Reply:
I guess this going to be $component->hasMethod($setter)